composables/storeFinder/useStoreLocations.ts

```diff
@@ -0,0 +1,37 @@
+import { ILocation } from "./useCurrentLocation"
+import { useState, useNuxtApp } from "#app"
+import { getSpLocations } from "~~/gql/queries/spLocations"
```

Comment:
use @ wherever is possible for consistency purpse

Reply:
Done